examples/lfric/eg20/Makefile

```diff
@@ -0,0 +1,99 @@
+# -----------------------------------------------------------------------------
```

Comment:
Oops, I should have noticed that. As discussed, this requires a change to the handling of the LFRicAlgorithmInvoke node so that it is not converted into a `call invoke_0` by the lowering process in the backend. That should be implemented as a transformatino instead.